client/package.json

```diff
@@ -42,6 +42,7 @@
     "react-dom": "^18.2.0",
     "react-draggable": "^4.4.5",
     "react-popper": "^2.3.0",
+    "react-use-websocket": "latest",
```

Comment:
do we want to pin to latest? might be good to use the specific version that is currenty working to spare us from headache if updates produce breaking changes

Reply:
Yeah you're right, changed that